apis/core/hydra/table.ttl

```diff
@@ -62,6 +62,18 @@ cc:Table
             <file:handlers/table/delete#remove> ;
         ] ;
     ] ,
+    # [
+    #   a
+    #     hydra:Operation,
+    #     schema:UpdateAction ;
+    #   hydra:title
+    #     "Edit table" ;
+    #   hydra:method
+    #     "PUT" ;
+    #   hydra:expects
+    #     cc:Table,
+    #     <shape/table/update> ;
+    # ],
```

Comment:
Maybe we can implement it already? It will fail if called but at least we can verify the UI bind correctly

Reply:
I already verified the UI bind. I commented it out to avoid users reporting bugs before the feature is actually implemented.